src/TimeSteppers/TimeSteppers.jl

```diff
@@ -108,52 +123,32 @@ end
 Calculate the (nonhydrostatic) pressure correction associated `tendencies`, `velocities`, and step size `Δt`.
 """
 function calculate_pressure_correction!(nonhydrostatic_pressure, Δt, tendencies, velocities, model)
-    velocity_tendencies = (u=tendencies.u, v=tendencies.v, w=tendencies.w)
+    velocity_tendencies = (u=model.timestepper.Gⁿ.u, v=model.timestepper.Gⁿ.v, w=model.timestepper.Gⁿ.w)
```

Comment:
This change means that `tendencies` is no longer needed in the function signature above?

Reply:
It's still necessary as it's passed to `solve_for_pressure!` a few lines below.

Reply:
ah, I missed that. 

We could put the line

```julia
tendency_data = datatuple(model.timestepper.Gⁿ)
```

in this function and remove the tendencies argument. But maybe we should hold off since we might refactor time-stepping soon anyways?

Reply:
Hmmm, yeah we'll probably be the next ones to refactor the time stepping code so might as well leave it as all the tests pass.